polymer-redux.js

```diff
@@ -54,18 +53,23 @@
                     value = Polymer.Base.get(path, state);
                 }
 
+                // prevent unnecesary polymer notifications
+                previous = element.get(property.name);
+                if(value === previous) {
+                    return;
```

Comment:
Is returning here the best option. I know data **SHOULD** be immutable, but what if the developer pushes an item to an `Array` or adds/removes a property to an `Object`. The reference will be the same and no updates will trigger.